### ISSUE TITLE
What is `manim_imports_ext`?

### ISSUE THREAD

**Issue by Gsparsh22** [3b1b/videos#108]
How to run code with `manim_imports_ext`?

**Comment by AlieNiT** [comment]
manim_imports_ext is the python file at the root directory of this repo ([here](https://github.com/3b1b/videos/blob/master/manim_imports_ext.py)).

manimgl always runs the [`universal_import_line`](https://github.com/3b1b/manim/blob/39fbb677dcc86fc32075249e66a3cbaba508aa7b/manimlib/default_config.yml#L174) at the begining of any run, which by default is `py from manimlib import *`.

However, the config can be changed using `custom_config.py` (apparently in the irectory in which you are running manimgl), and Grant has done that. He has changed `universal_import_line` to be "from manim_imports_ext import *" (checkout [here](https://github.com/3b1b/videos/blob/7ae22f3912eb775e3f4b7c03a3efc54cde746994/custom_config.yml#L11)). Therefore, manimgl tries to run this line of code in python which can cause error if PYTHONPATH is not set to be the root directory of this repo (I guess there are other ways, but it's the one I know about.). Because python should know where to look for this file.

Long story short, before running manimgl, you should set the PYTHONPATH to be repos root directory, or simply add `PYTHONPATH=.` before manimgl when running it (e.g. run `PYTHONPATH=. manimgl`).

I should also point out that to the best of my knowledge, this the way that the whole repo works, so if you just comment out the import line in the config, many other codes won't run because they also assume PYTHONPATH is the root directory of the project.

**Comment by Gsparsh22** [comment]
okk, had a different PYTHONPATH environment variable other than the repo root, so had an error. Thanks for helping
